src/hexen/h2_main.c

```diff
@@ -676,6 +676,23 @@ static void HandleArgs(void)
         ST_Message("Playing demo %s.\n", myargv[p+1]);
     }
 
+    //!
+    // @category demo
+    //
+    // Record or playback a demo with high resolution turning.
```

Comment:
One thing: we can't avoid needing the user to specify `-longtics` when recording demos. But is it possible to avoid needing it when playing back? For Doom we use a different demo version (matching the "Doom 1.91" hack). Any chance we can do the same for Heretic and Hexen too?

One thing to check might be the Heretic+ / Hexen+ that Entryway (PrBoom+ developer) has made: do these also record longtics demos? If so then perhaps we can reuse the version numbers.

https://sourceforge.net/projects/prboom-plus/files/